src/databricks/labs/remorph/config.py

```diff
@@ -96,3 +96,27 @@ class TranspilationResult:
 class ValidationResult:
     validated_sql: str
     exception_msg: str | None
+
+
+@dataclass
+class ReconcileTablesConfig:
+    filter_type: str  # all/include/exclude
+    tables_list: list[str]  # [*, table1, table2]
+
+
+@dataclass
+class ReconcileConfig:
+    __file__ = "reconcile_config.yml"
```

Comment:
```suggestion
    __file__ = "reconcile.yml"
```
it _might be better_ to name it like this - .e.g it's already a config

Reply:
Done